distribution/lib/Standard/Database/0.0.0-dev/src/Internal/JDBC_Connection.enso

```diff
@@ -138,8 +138,9 @@ create : Text -> Vector -> JDBC_Connection
 create url properties = handle_sql_errors <|
     java_props = Properties.new
     properties.each pair->
-        if pair.second.is_nothing.not then java_props.insert pair.first pair.second else
-            java_props.remove pair.first
+        case pair.second of
+            Nothing -> Polyglot.invoke java_props "remove" [pair.first]
+            _ -> Polyglot.invoke java_props "setProperty" [pair.first, pair.second]
```

Comment:
I am glad the `Polyglot.invoke` works and workarounds the _perception_ of `java.util.Properties` being a `Map`.